ops/model.py

```diff
@@ -218,23 +219,22 @@ def __getitem__(self, relation_name):
         relation_list = self._data[relation_name]
         if relation_list is None:
             relation_list = self._data[relation_name] = []
-            relation_role = self._relations_meta[relation_name].role
+            our_app = self._our_unit.app if self._relations_meta[relation_name].role == 'peers' else None
```

Comment:
Related to the first comment. I suggest avoiding ternary operators in such cases.